You can produce the issue like this. This is `file.py`

```
from sqids import Sqids
_ = Sqids()
```

Then run:

```
$ mypy --strict file.py
file.py:1: error: Module "sqids" does not explicitly export attribute "Sqids"  [attr-defined]
$ pyright file.py
/home/user/file.py
  /home/user/file.py:1:19 - error: "Sqids" is not exported from module "sqids"
    Import from "sqids.sqids" instead (reportPrivateImportUsage)
1 error, 0 warnings, 0 informations
```

`__all__ = ["Sqids"]` does also solve the issue. I just prefer the `from sqids import Sqids as Sqids` way, since it looks better in a file with tons of exports and it is simpler to append another export and keep them sorted with `reorder-python-imports`/`isort`. But `__all__ = ["Sqids"]` is probably more common. Something about that String notation just rubs me the wrong way.

--- a/sqids/__init__.py
+++ b/sqids/__init__.py
@@ -1,1 +1,3 @@
-from .sqids import Sqids  # noqa: F401
+from .sqids import Sqids
+
+__all__ = ["Sqids"]

